### PR TITLE
add Julia Programming language to footer

### PR DESCRIPTION
**Pull request by vchuravy**
cc: @aviks

### CODE REVIEW

File: app/views/home/about.html.erb [hunk 1]
```diff
@@ -27,7 +27,7 @@
 
         <h1>Submissions</h1>
 
-        <p>Not all papers are eligible to be published in the JuliaCon proceedings. Submissions must be related to JuliaCon contributions such as talks, posters, or similar. Generally, JCON is accepting two different kinds of submissions:
+        <p>Not all papers are eligible to be published in the JuliaCon proceedings. Submissions must be related to JuliaCon contributions such as talks, posters, or similar. Generally, JuliaCon is accepting two different kinds of submissions:
```
Comment (aviks) [inline app/views/home/about.html.erb:30]:
```suggestion
        <p>Not all papers are eligible to be published in the JuliaCon proceedings. Submissions must be related to JuliaCon contributions such as talks, posters, or similar. Generally, this journal will accept two different kinds of submissions:
```

**Comment by arfon** [comment]
Is this good to merge?

**Comment by vchuravy** [comment]
It is!

**Comment by arfon** [comment]
This is looking a little weird?

Screen Shot 2021-02-15 at 20 36 53

**Pull request merged**